app/app.rb

```diff
@@ -0,0 +1,83 @@
+RACK_ENV ||= 'development'
+
+require 'sinatra/base'
+require 'sinatra/flash'
+require_relative 'data_mapper_setup'
+
+class Chitter < Sinatra::Base
+
+  enable :sessions
+  register Sinatra::Flash
+  set :session_secret, 'super secret'
+  use Rack::MethodOverride
+
+
+  get '/' do
+    'Hello Chitter!'
+  end
+
+  get '/users/new' do
+    @user = User.new
+    erb :'users/new'
+  end
+
+  post '/users' do
+    @user = User.new(name: params[:name], username: params[:username], email: params[:email], password: params[:password], password_confirmation: params[:password_confirmation])
```

Comment:
Metrics/LineLength: Line is too long. [177/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)